lib/composites/tiles/tiles.less

```diff
@@ -1,13 +1,88 @@
 @import "../../variables.less";
 
 @{prefix}tile {
-  background-color: @tile-bg;
-  .vendor(box-shadow, 0px 0px 13px 1px rgba(0,0,0,0.39));
-  .border-box();
-  position: relative;
-
-  &@{prefix}small {
-    width: 298px;
-    margin: 0 auto;
+  .tile-drop();
+  max-width: 476px;
```

Comment:
UX said they need to play around with widths for tiles, so once they make a final decision I will store this value in a variable